### ISSUE TITLE
tornadoes() URL is out of date

### ISSUE THREAD

**Issue by mbjoseph** [ropensci/rnoaa#311]
It seems like the NOAA URL used for the `rnoaa::tornadoes()` function (https://github.com/ropensci/rnoaa/blob/master/R/tornadoes.R#L32) is no longer valid: https://www.spc.noaa.gov/gis/svrgis/zipped/tornado.zip

The tornado paths now seem to be here: https://www.spc.noaa.gov/gis/svrgis/zipped/1950-2017-torn-aspath.zip

``` r
library(rnoaa)

tornadoes()
#> Error: Forbidden (HTTP 403)
```

<sup>Created on 2019-09-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

I'll cook up PR that provides a fix and adds tests for `rnoaa::tornoadoes()`, related to #208.

<details> <summary><strong>Session Info</strong></summary>

```r

─ Session info ───────────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.6.1 (2019-07-05)
 os       Ubuntu 18.04.3 LTS          
 system   x86_64, linux-gnu           
 ui       RStudio                     
 language en_US                       
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/Denver              
 date     2019-09-06                  

─ Packages ───────────────────────────────────────────────────────────────────────────────────────
 package     * version date       lib source        
 assertthat    0.2.1   2019-03-21 [1] CRAN (R 3.6.0)
 callr         3.3.1   2019-07-18 [1] CRAN (R 3.6.1)
 cli           1.1.0   2019-03-19 [1] CRAN (R 3.6.0)
 clipr         0.7.0   2019-07-23 [1] CRAN (R 3.6.1)
 codetools     0.2-16  2018-12-24 [4] CRAN (R 3.5.2)
 crayon        1.3.4   2017-09-16 [1] CRAN (R 3.6.0)
 digest        0.6.20  2019-07-04 [1] CRAN (R 3.6.1)
 evaluate      0.14    2019-05-28 [1] CRAN (R 3.6.0)
 fs            1.3.1   2019-05-06 [1] CRAN (R 3.6.0)
 htmltools     0.3.6   2017-04-28 [1] CRAN (R 3.6.0)
 knitr         1.24    2019-08-08 [1] CRAN (R 3.6.1)
 lattice       0.20-38 2018-11-04 [4] CRAN (R 3.5.1)
 processx      3.4.1   2019-07-18 [1] CRAN (R 3.6.1)
 ps            1.3.0   2018-12-21 [1] CRAN (R 3.6.0)
 R6            2.4.0   2019-02-14 [1] CRAN (R 3.6.0)
 raster      * 3.0-2   2019-08-22 [1] CRAN (R 3.6.1)
 Rcpp          1.0.2   2019-07-25 [1] CRAN (R 3.6.1)
 reprex        0.3.0   2019-05-16 [1] CRAN (R 3.6.0)
 rlang         0.4.0   2019-06-25 [1] CRAN (R 3.6.1)
 rmarkdown     1.15    2019-08-21 [1] CRAN (R 3.6.1)
 rstudioapi    0.10    2019-03-19 [1] CRAN (R 3.6.0)
 sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.6.0)
 sp          * 1.3-1   2018-06-05 [1] CRAN (R 3.6.0)
 whisker       0.4     2019-08-28 [1] CRAN (R 3.6.1)
 withr         2.1.2   2018-03-15 [1] CRAN (R 3.6.0)
 xfun          0.9     2019-08-21 [1] CRAN (R 3.6.1)

[1] /home/max/R/x86_64-pc-linux-gnu-library/3.6
[2] /usr/local/lib/R/site-library
[3] /usr/lib/R/site-library
[4] /usr/lib/R/library
```
</details>

**Comment by sckott** [comment]
ugh, and thanks for pointing this out. thanks for PR'ing

**Comment by sckott** [comment]
fixed via #312
